### ISSUE TITLE
[a11y-defect-3]: Improve edit labels on "Review the evidence you’re submitting" page

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#101973]
### Point of contact

@davidakennedy

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

The Edit labels for the "Review the evidence you’re submitting" page, while good, could be better.

<img width="720" alt="Red arrows pointing to the edit links listed in this issue." src="https://github.com/user-attachments/assets/e2eed557-394e-461f-bb49-942629aa4d81" />

I'd advocate for them aligning to the bold text near by, as some already do.

### Reproduction steps

1. Log in with a test user.
2. Visit: https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/
3. Fill out the form until you get to: https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/summary
4. View labels on edit links.

### Proposed solution or next steps

Recommendation:

```html
aria-label="edit authorization choice"
```
becomes:
```html
aria-label="Edit authorize the release of non-VA medical records to the VA (21-4142)"
```

```html
aria-label="Edit do you want to limit information"
```
becomes
```html
aria-label="Edit do you want to limit the information we can request?"
```

```html
aria-label="Edit limitations"
```
becomes
```html
aria-label="Edit what do you want your information request to be limited to?"
```

This matches the content elsewhere, and makes the labels more clear.

### References, articles, or WCAG support

N/A

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [x] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by Mottie** [comment]
Changes are now available in staging for validation.

**Comment by eileen-coforma** [comment]
@davidakennedy will review

**Comment by davidakennedy** [comment]
I gave this a look, and the labels are changed and improved, as outlined in the ticket. This can be closed as complete.

cc: @comaurice 

![Screenshot showing the HTML of the page, which the corrections made, as outlined in the ticket.](https://github.com/user-attachments/assets/4d582581-6d8d-4cda-ba4f-1e0ded9978d0)
